### PR TITLE
[1.16] bump to  critools 1.16

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@umohnani8 @saschagrunert @mrunalp PTAL

**Comment by umohnani8** [comment]
@haircommander do you know if this fixes https://github.com/kubernetes-sigs/cri-tools/issues/548?

**Comment by haircommander** [comment]
these are work arounds for at least one of the problems reported. I think those tests have to be updated

**Comment by saschagrunert** [comment]
Then we can forward port this one and close my open PR, thanks! 🙏😊

**Comment by haircommander** [comment]
/retest 
(apparently, if you open a PR using the same commit but different branch, it effects both PRs)

File: test/ctr.bats [hunk 1]
```diff
@@ -212,7 +212,7 @@ function teardown() {
 	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run crictl rm "$ctr_id"
```
Comment (TomSweeneyRedHat) [inline test/ctr.bats]: Sorry if a dupe comment, having fun network issues.  Although very minimal, I'd really rather not see test changes in a version bump PR unless they would break due to the bump.  I think these should be in a separate PR.

Comment (haircommander) [reply test/ctr.bats]: I got this change from @saschagrunert's PR. Thought it was needed, though never checked. Let's find out :smile:

Comment (haircommander) [reply test/ctr.bats]: yes, it appears without the change the tests break

Comment (TomSweeneyRedHat) [reply test/ctr.bats]: Thanks for checking @haircommander, I was fine with them in, more of a reminder for the future.  But saying that, I'm puzzled that it's required, I would not have guessed that.

**Comment by haircommander** [comment]
Finally! Can i have that lgtm back?
